packages/cli/src/databases/postgresdb/migrations/1636626154934-CreateUserManagement.ts

```diff
@@ -0,0 +1,146 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+import { v4 as uuid } from 'uuid';
+import config = require('../../../../config');
+import { loadSurveyFromDisk } from '../../utils/migrationHelpers';
+
+export class CreateUserManagement1636626154934 implements MigrationInterface {
+	name = 'CreateUserManagement1636626154934';
+
+	public async up(queryRunner: QueryRunner): Promise<void> {
+		const tablePrefix = config.get('database.tablePrefix');
+		await queryRunner.query(
+			'CREATE TABLE "' + tablePrefix + 'role" (' +
+				'"id" serial NOT NULL,' +
+				'PRIMARY KEY ("id"),' +
+				'"name" VARCHAR(32) NOT NULL,' +
+				'"scope" VARCHAR(250) NOT NULL,' +
+				'"createdAt" timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,' +
+				'"updatedAt" timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,' +
+				'CONSTRAINT "UQ_' + tablePrefix + '5b49d0f504f7ef31045a1fb2eb8" UNIQUE ("scope", "name")' +
```

Comment:
i think we should rather go for the `UNIQUE INDEX` here because we are also querying for this combination 👍🏼